Compilation issue with netcdf-4.9.3

Compiling against netcdf-4.9.3 results in this compile failure:

/data/users/david.davies/sources/bufr-query/core/include/bufr/encoders/netcdf/NetcdfHelper.h: In function ‘void bufr::encoders::netcdf::addAttribute(netCDF::NcVar&, const string&, const T&) [with T = std::__cxx11::basic_string<char>; std::string = std::__cxx11::basic_string<char>]’:
/data/users/david.davies/sources/bufr-query/core/include/bufr/encoders/netcdf/NetcdfHelper.h:56:19: error: ‘_FillValue’ was not declared in this scope; did you mean ‘NC_FillValue’?
   56 |       if (name != _FillValue)
      |                   ^~~~~~~~~~
      |                   NC_FillValue

Seems to be related to https://github.com/Unidata/netcdf-c/issues/3029.

--- a/core/src/encoders/netcdf/Encoder.cpp
+++ b/core/src/encoders/netcdf/Encoder.cpp
@@ -147,7 +147,7 @@ namespace netcdf {
           var.setCompression(true, true, compressionLevel);
         }
 
-        addAttribute(var, _FillValue, obj->missingValue());
+        addAttribute(var, FillValueStr, obj->missingValue());
         obj->write(std::make_shared<VarWriter<T>>(var));
 
         return var;
@@ -307,7 +307,7 @@ namespace netcdf {
             {
                 const auto& ncDim = file->addDim(dim->dimObj->name, dim->dimObj->size());
                 auto ncVar = file->addVar(dim->dimObj->name, nc::NcType::nc_INT, ncDim);
-                addAttribute(ncVar, _FillValue, DataObject<int>::missingValue());
+                addAttribute(ncVar, FillValueStr, DataObject<int>::missingValue());
                 dim->dimObj->write(std::make_shared<VarWriter<int>>(ncVar));
             }
 

